src/code/models/node.ts

```diff
@@ -77,7 +82,7 @@ export class Node extends GraphPrimitive {
   constructor(nodeSpec, key?, isTransfer?) {
     super(isTransfer ? "Transfer" : "Node");
 
-    let val, val1, val10, val11, val2, val3, val4, val5, val6, val7, val8, val9;
+    let val, val1, val10, val11, val12, val2, val3, val4, val5, val6, val7, val8, val9;
```

Comment:
Well, 😕 I don't love this. 

I realize this must be the result of some automatic conversion, but ... Seems like we at least need to put a PT story in to fix it.